src/SecurityChecker.php

```diff
@@ -121,15 +121,20 @@ protected function fetchAdvisoryComposerJson($url)
      *   - version
      *   - links.
      */
-    protected function calculateSecurityUpdates(array $composer_lock_data, array $security_advisories_composer_json)
+    protected function calculateSecurityUpdates(array $composer_lock_data, array $security_advisories_composer_json, bool $excludeDev = false)
     {
+        if ($excludeDev) {
+            $packages = $composer_lock_data['packages-dev'];
```

Comment:
It seems to me that with it will check only dev dependencies with that line: `packages` should be used instead of `packages-dev`, no?